meta_package/global_variables.bzl

```diff
@@ -16,3 +16,12 @@ LOGGING_LIB_DEFINE = select({
     "@bazel_tools//src/conditions:windows": ["R_USE_GLOG"],
     "//conditions:default": ["R_USE_BOOST_LOG"],
 })
+
+WINDOWS_BUILD_DEFINES = [ # use it later
+    "-DR_PUBLIC_API=__declspec(dllexport)",
+]
+
+PUBLIC_API_PREFIX = select({ # use it later
```

Comment:
Maybe we should use it now, since it will save us a lot of trouble.

Reply:
I'll do it in a separate PR, ok?

Reply:
I think we should do it now, it will allow for less changes